apps/extension/src/ui/domains/Portfolio/AssetsTable/usePortfolioSymbolBalances.ts

```diff
@@ -93,11 +94,11 @@ const sortSymbolBalancesBy =
 
 export const usePortfolioSymbolBalances = (balances: Balances) => {
   const currency = useSelectedCurrency()
+  const [hideDust] = useSetting("hideDust")
 
   // group balances by token symbol
   // TODO: Move the association between a token on multiple chains into the backend / subsquid.
   // We will eventually need to handle the scenario where two tokens with the same symbol are not the same token.
-  // Also, we might want to separate testnet tokens from non-testnet tokens.
```

Comment:
Since writing that comment we've added this separation between testnet and non-testnet tokens.